### PR TITLE
use clarify-error on all public APIs

### PR DESCRIPTION
**Pull request by staltz**

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -365,16 +361,11 @@ exports.init = function (sbot, config) {
         } else if (SSBURI.isBendyButtV1FeedSSBURI(msgVal.author)) {
           const previous = (state[msgVal.author] || { value: null }).value
           const err = bendyButt.validateSingle(msgVal, previous, hmacKey)
-          if (err) {
-            debug(
-              `validation failed for bendy butt message in addImmediately(): ${err.message}`
-            )
-            return cb(err)
-          }
```
Comment (staltz) [inline db.js:373]: With the new changes, I was getting a bunch of lines formatted like this under `prettier`, and it was getting hard to follow the "normal" (non-error) control flow, so I decided to turn off prettier for these lines that handle errors with clarify-error

File: compat/history-stream.js [hunk 2]
```diff
@@ -62,11 +63,13 @@ exports.init = function (sbot, config) {
             sbot.db
               .getJITDB()
               .paginate(query, 0, limit, false, false, (err, answer) => {
-                cb(err, pull.values(answer.results.map(formatMsg)))
+                if (err) cb(clarify(err, 'ssb-db2 createHistoryStream failed')) // prettier-ignore
```
Comment (arj03) [inline compat/history-stream.js:66]: This one is exposed over the network. Can that cause trouble that we might leak information about the local system? I checked compat/ebt and that is fine as getvectorclock is only used internally.

File: db.js (hunk 1 above)
Comment (arj03) [reply db.js:373]: I like the shorter lines here. And I guess not doing debug is fine, as we'll catch these errors further up the stack.

**Review by arj03 — APPROVED** [review]
I look forward to seeing errors (ohh that sounds so wrong :-P) after all of these modules have been upgraded to clarify error
